src/main.c

```diff
@@ -307,20 +307,22 @@ static int read_authkey()
 	int fd;
 
 	booth_conf->authkey[0] = '\0';
-	if (stat(booth_conf->authfile, &booth_conf->authstat) < 0) {
-		log_error("cannot stat authentication file %s: %s",
+	fd = open(booth_conf->authfile, O_RDONLY);
```

Comment:
You mean that there's a time between stat and open for the file to change?

Reply:
For whatever item in the traversal path to change (e.g. `/etc` turned into symlink).
As said hypothetical, but good to follow secure practices and consequently
suppress the alerts.

Reply:
ok

Reply:
thanks!